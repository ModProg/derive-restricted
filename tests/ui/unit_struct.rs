#[derive_where::derive_where(Clone)]
struct Test();

fn main() {}
