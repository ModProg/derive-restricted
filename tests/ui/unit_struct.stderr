error: derive-where doesn't support items without generics, as this can already be handled by standard `#[derive()]`.
 --> tests/ui/unit_struct.rs:2:1
  |
2 | struct Test();
  | ^^^^^^^^^^^^^^
